File: Pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height; //declare int variable height
    do
    {
        height = get_int("height: ");
    }
    while (height < 1 || height > 8); // height should not be less than 1 or greater than 8.
    for (int row = 0; row < height; row++) //print new line as row.
    {
        for (int space = height - row - 1; space > 0; space--) // to print space
        {
            printf(" ");
        }
        for (int hash = 0; hash < row + 1; hash++) //to print hashes(#)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: Pset6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break
for row in range(height):
    for space in range(height - row - 1, 0, -1):
        print(" ", end="")
    for hash in range(row + 1):
        print("#", end="")
    print("  ", end="")
    for right_hash in range(row+1):
        print("#", end="")
    print("\n", end="")